### DISCUSSION TITLE
Thank you wulfy

### DISCUSSION THREAD

**Discussion by neilo1** [wulfy23/rpi4#46] [General]
Thank you wulfy for coming back and making a new build for our pi4 routers. 🤗

**Comment by molly78** [comment]
Seconded - huge thanks!

**Comment by wulfy23** [comment]
you are most welcome, and it's a relief to see that we did not really miss too much over all that time (in a general sense).

3 weeks-ish late for 22.03.6? if i'd built it and skipping 23.05.0 and 23.05.1? I probably would have done anyway, but yeah, thought I was like 4 months / 2 major releases behind...

either way, hopping over to nft base was kinda 'without much testing' and introduces a large chance for 'config staleness' and related glitches, so if any of you guys were able to upgrade and ~ most ~ of your stuff worked then that's credit to the upstream guys and the help I got here from you testing the last snapshot or two...

apologies I could't be more active but I'm not really following development these days, so I just kinda put out fires when they pop up and make sure the images build, upgrade and 'mostly' work...

i see the pi5 came out lol (don't think i'll be rushing to get one but yeah with all the criticisms of the pi4 it sure has been outstanding in it's service and properties in this router function)... 

you know, all the things that I started this build for have pretty much been addressed upstream accept for the usb / larger default partition sizes possibilities (afaik there was talk about a patch for custom sizes with ASU )...

but yeah, with all the time i put into it would be a shame to just let that code slowly sink deeper into my archive backups, never to be useful again ;) so with luck I can get ride of a bit more junk and minimum see 23.05.x till EOL. ( Fun fact... I got an N5105 and N100, but it just ain't the same ) the automations here are just too damn useful and familiar to me...

SIDENOTE: didn't update the 'boot elf dat' files tho' we were probably due for that... so will run a test this weekend... have not heard much blowback so seems like no (major) drama there with established hardware...
